app/forms/generic_asset_form.rb

```diff
@@ -28,6 +28,7 @@ def template
   def template_terms
     template.visible_property_names
   end
+
```

Comment:
Guess when comments are on a commit they don't show up properly with context :-/

This and the one below seem like random additions that will potentially cause merge conflicts with no real value